components/admin_console/license_settings/license_settings.jsx

```diff
@@ -135,6 +140,7 @@ export default class LicenseSettings extends React.PureComponent {
 
         await this.props.actions.getLicenseConfig();
         this.setState({fileSelected: false, fileName: null, serverError: null, removing: false});
+        this.props.actions.getPrevTrialLicense();
```

Comment:
combine `this.props.actions.getLicenseConfig()` & `this.props.actions.getPrevTrialLicense()` into promise all and use an `await`?

Reply:
@BenCookie95 we don't need to use promise.all here but I have moved `getPrevTrialLicense` before `getLicenseConfig` so `getLicenseConfig` runs in background without affecting the way we're using `getLicenseConfig` right now.